packages/twenty-front/src/modules/ui/input/button/components/Button.tsx

```diff
@@ -194,7 +194,7 @@ const StyledButton = styled.button<
               box-shadow: ${!disabled && focus
                 ? `0 0 0 3px ${theme.color.red10}`
                 : 'none'};
-              color: ${!disabled ? theme.font.color.danger : theme.color.red20};
+              color: ${!disabled ? theme.color.red : theme.font.color.danger};
```

Comment:
@Bonapara could you provide the value of the red disabled button color for dark mode. I cannot find it in the Figma :(

For the light one, I see that you are actually not changing the font but changing the overall opacity which could be an issue depending what's behind the button. is this intended?

Reply:
![Uploading image.png…]()

Reply:
<img width="1274" alt="image" src="https://github.com/twentyhq/twenty/assets/12035771/764a32e1-0336-431c-a239-0f1e875c7937">

Reply:
Per discussion with Bonapara, we should update Button and IconButton to have same font as non disabled but opacity to 0.24%

Reply:
Ok. What I did was follow the same color design being used by the border's light and dark variant in disabled mode. since, button's border is working fine as expected.

I was trying to use the existing values without introducing any new color values which basically work great and is tested as well on multiple pages with dark and light mode.

Reply:
> @Bonapara could you provide the value of the red disabled button color for dark mode. I cannot find it in the Figma :(
> 
> For the light one, I see that you are actually not changing the font but changing the overall opacity which could be an issue depending what's behind the button. is this intended?

<img width="765" alt="image" src="https://github.com/twentyhq/twenty/assets/60315832/07bf49db-a787-42c5-9f23-e910f504d691">

I don't think there is an opacity added to the color. the colors red20 and red70 do not have any opacity as I can see.